google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ChecksumResultSetTest.java

```diff
@@ -25,18 +25,11 @@
 import com.google.cloud.ByteArray;
 import com.google.cloud.Date;
 import com.google.cloud.Timestamp;
-import com.google.cloud.spanner.AbortedDueToConcurrentModificationException;
-import com.google.cloud.spanner.AbortedException;
-import com.google.cloud.spanner.ResultSet;
-import com.google.cloud.spanner.ResultSets;
+import com.google.cloud.spanner.*;
```

Comment:
nit: Google code style forbids the use of wildcard imports (https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
fixed.